### PR TITLE
FLUID-5927: Asserting that context are the same DOM element

### PR DESCRIPTION
**Pull request by jobara**
Doing a deep equals on the documents causes Safari to fail as there are some lower level properties
that aren't set yet and have TyperError as a value. E.g. selectionStart. However it should be
sufficient to do an assertEquals to ensure that the contexts refer to the same Document.

https://issues.fluidproject.org/browse/FLUID-5927

### CODE REVIEW

File: tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js [hunk 1]
```diff
@@ -347,7 +347,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             fluid.each(memberNames, function (memberName) {
                 var comp = parentPanel[memberName];
                 jqUnit.assertNotUndefined("The " + memberName + " should be instantiated", comp);
-                jqUnit.assertDeepEq("The context for " + memberName + " should be retained", comp.origContext, comp.container[0].ownerDocument);
```
Comment (amb26) [inline tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js]: This change is fine, and the tests should never have been using assertDeepEq for DOM nodes. However, I can't test this pull myself.

Comment (colinbdclark) [inline tests/framework-tests/preferences/js/SeparatedPanelPrefsEditorTests.js]: I ran this test in Justin's branch on my Mac and it now passes with no failure messages in Safari.

**Comment by cindyli** [comment]
The fix on the same issue has been addressed and merged in [this pull request](https://github.com/fluid-project/infusion/pull/734). @jobara, can you please verify? Thanks.

**Comment by jobara** [comment]
@cindyli yes thanks for catching that. I knew I felt like I had fixed that before.

**Comment by jobara** [comment]
Fixed by https://github.com/fluid-project/infusion/pull/734

**Pull request closed without merging**